workflows/update_tags.sh

```diff
@@ -25,28 +25,37 @@
 # This is necessary since a new docs-csm version could land before or after a CSM update.
 
 set -e
-REGISTRY="registry.local"
+REGISTRY_NAME="registry.local"
+REGISTRY_PROTOCOL="https"
+PODMAN_TLS=""
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 cd $SCRIPT_DIR
 
+
+# Ensure the script is running from within the workflows directory so it finds argo yaml files."
 if [[ "$(basename ${SCRIPT_DIR})" != "workflows" ]]; then
   echo "ERROR: This script must be located in the workflows directory to run."
   exit 1
 fi
 
-if [[ $(nslookup $REGISTRY >> /dev/null; echo $?) != 0 ]]; then
-  echo "INFO: Skipping update_tags.sh script given docs were installed outside of a CSM environment."
-  exit 0
+# Change registry URL target for PIT
+if [[ -f /etc/pit-release ]]; then
+  echo "INFO: Running on PIT. Updating registry URL to $REGISTRY_NAME."
```

Comment:
Whether we use `registry.local` or `pit.nmn:5000` doesn't really depend on where the script is running.   This can run on m002 or w003 when `registry.local` is not available and it can run on the pit when `pit.nmn:5000` is not available.  Instead, I would try `registry.local` and if that fails then try `pit.nmn:5000`.   And if that fails, then you print out the warning.

Reply:
Or you could skip `pit.nmn:5000` altogether if there is something that will be updating the tags later when CSM services are installed.

Reply:
done